public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,261 @@
+      summary: Get a balance
+      description: Get the balance associated with a guarantee reference number (GRN).
+      operationId: getGuaranteeBalance
+      parameters:
+        - $ref: '#/components/parameters/acceptHeader'
+        - $ref: '#/components/parameters/contentTypeHeader'
+        - in: path
+          name: grn
+          schema:
+            type: string
+            pattern: '^[0-9]{2}[A-Z]{2}[A-Z0-9]{12}[0-9]([A-Z][0-9]{6})?$'
+          required: true
+          description: The guarantee reference number (GRN) of the guarantee.
+      requestBody:
+        description: ''
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/CustomsGuaranteesBalancesrequest'
+                - example:
+                    accessCode: ABC1
+            example:
+              accessCode: ABC1
+        required: true
+      responses:
+        '200':
+          description: A successful balance request response
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/GetBalanceOk'
+                  - example:
+                      balance: 12345678.90
+              example:
+                _links:
+                  self:
+                    href: /customs/guarantees/23GB0123456789AB2/balance
+                balance: 12345678.90
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/BadRequest'
+                  - description: The supplied guarantee reference number or JSON payload was invalid.
+                    example:
+                      code: BAD_REQUEST
+                      message: The access code was not supplied.
+              example:
+                code: BAD_REQUEST
+                message: The access code was not supplied.
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/NotFound'
+                  - description: The guarantee reference number or access code did not match an existing guarantee.
+                    example:
+                      code: NOT_FOUND
+                      message: Guarantee balance not found.
+              example:
+                code: NOT_FOUND
+                message: Guarantee balance not found.
+        '406':
+          description: Not Acceptable
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/AcceptHeaderInvalid'
+                  - example:
+                      code: ACCEPT_HEADER_INVALID
+                      message: The Accept header must be set to application/vnd.hmrc.2.0+json to use this resource.
```

Comment:
I would be inclined to add back ticks here.

`Accept` header
`application/vnd.hmrc.2.0+json`

Reply:
No - this is a response from the service and that won't include backticks.

Reply:
OK.